### PR TITLE
service url from ENV

### PR DESCRIPTION
**Pull request by bluta**

### CODE REVIEW

File: lib/editorium/config.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Configuration
     attr_accessor :service_url, :preview_layout
 
     def initialize
-      @service_url = 'http://editorium2.herokuapp.com'
```
Comment (bluta) [inline lib/editorium/config.rb:15]: use the old ( --1 )

**Review by bluta — COMMENTED** [review]
get back to editorium.herokuapp.com

**Comment by bluta** [comment]
[#2049 editorium for china](https://trello.com/c/XoTFA2lO/2049-2049-editorium-for-china)

File: lib/editorium/config.rb (hunk 1 above)
Comment (balazslaszlo) [reply lib/editorium/config.rb:15]: done

**Comment by balazslaszlo** [comment]
[#2039 editorium on http](https://trello.com/c/mb2YUs6n/2039-2039-editorium-on-http)

**Pull request merged**